conda/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "jina-hubble-sdk" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jina-hubble-sdk-{{ version }}.tar.gz
+  sha256: bab9b7b4d8c5965e099e07effd92cc0607b56eead0b4d26d6e5b473cb93daf8f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools >=18.0
+    - wheel
+    - pip
+  run:
+    - python >=3.7
+    - requests ==2.27.1
+    - aiohttp ==3.8.1
+    - rich >=12.0.0
```

Comment:
Is this mandantory because we already have `requirements.txt`, maintaining them in two places is hard

Reply:
This is not mandatory. We just recorded the information of the first version of meta.yaml, we don't need to keep track of it.
And [conda-forge/staged-recipes](https://github.com/conda-forge/staged-recipes) will auto trace the version of jina-hubble-sdk after conda-forge merge this https://github.com/conda-forge/staged-recipes/pull/20927